lib/pronto/clients/bitbucket_server_client.rb

```diff
@@ -0,0 +1,64 @@
+class BitbucketServerClient
```

Comment:
Could parts of this class be extracted to reduce duplication between `BitbucketServerClient` and `BitbucketClient`?